apps/frontend/.dockerignore

```diff
@@ -8,6 +8,7 @@
 !tsconfig.json
 !iis.conf
 !nginx.conf
```

Comment:
If nginx.conf is being removed/renamed, can it also be removed from the dockerignore

Reply:
Thank you. Removed.